src/esp/assets/PTexMeshData.cpp

```diff
@@ -311,8 +442,20 @@ void PTexMeshData::loadMeshData(const std::string& meshFile) {
   submeshes_.clear();
   if (splitSize_ > 0.0f) {
     LOG(INFO) << "Splitting mesh... ";
-    submeshes_ = splitMesh(originalMesh, splitSize_);
-    LOG(INFO) << "done" << std::endl;
+
+    // In this version, we load the sorted faces directly from an external
+    // binary file dumped out from ReplicaSDK, and disable the function
+    // splitMesh(...)
+
+    // See detailed comments in front of the splitMesh(...)
+    std::string subMeshesFilename = Corrade::Utility::Directory::join(
+        atlasFolder_, "../habitat/sorted_faces.bin");
+    submeshes_ = loadSubMeshes(originalMesh, subMeshesFilename);
+
+    // TODO:
```

Comment:
Is this comment still relevant?

Reply:
Yes, this TODO is relevant. It reminds us that the code introduced by this PR is a temporary solution.
The ideal solution is to call the splitMesh(originalMesh, splitSize_), and do the computation on the fly. But it replies on FRL lab, who needs to fix the Replica dataset first.
When that day comes, one can just "revert this PR".